types/amcharts/index.d.ts

```diff
@@ -1,3495 +1,137 @@
-// Type definitions for amCharts
+// Type definitions for amCharts 3.21
```

Comment:
since this is a technically a major version upgrade, can you move the old version into a different folder. see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

Reply:
Hi @mhegazy 
thanks for your feedback, but due to this one shipped already with #22150 and I am completly unaware, which version was previously supported, I couldn't even tell, if this is a major update. The added version number follows improvement suggestions from @micksatana and should provide information which version is supposed to be supported by now and in future. If from now on major version changes come, I will follow your instructions.
For adding the version number to the unknown state, I hope it is ok, to not have the unknown state any longer?